apps/coscrad-frontend/src/configurable-routing.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { screen } from '@testing-library/react';
+import { RouterProvider } from 'react-router-dom';
+import { createAppRouter } from './app/create-app-router';
+import { renderWithProviders } from './utils/test-utils';
+import { getDummyConfigurableContent } from './utils/test-utils/get-dummy-configurable-content';
+
+const dummyContentConfig = getDummyConfigurableContent();
+
+describe('custom configured routing', () => {
```

Comment:
Is this test still being used? Is it complete? It seems redundant with the other unit \ integration tests of dynamic routes.

Reply:
Removed.